app/helpers/application_helper.rb

```diff
@@ -159,19 +159,6 @@ def auto_complete_controller_name
     controller.respond_to?(:auto_complete_controller_name) ? controller.auto_complete_controller_name : controller_name
   end
 
-  def auto_complete_search(name, val, options = {})
```

Comment:
This is still used in https://github.com/theforeman/foreman_remote_execution/blob/972e8536f602068445527b3813609fa543694851/app/views/job_templates/_custom_tabs.html.erb#L4
I'll try to do something about that before merging :)

Reply:
https://github.com/theforeman/foreman_remote_execution/pull/672